### PR TITLE
FileService Class

### PR DESCRIPTION
**Pull request by AliAla**
Read and write method

### CODE REVIEW

File: src/main/FileService.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.Scanner;
+import javax.swing.JOptionPane;
+
+/**
+ * @author alial
+ *
+ */
+public class FileService {
+
+	String fileName = "jMoSS.txt";
```
Comment (rmit-s3675667-raj-shrestha) [inline src/main/FileService.java]: Add `private` to protect the attribute

File: src/main/FileService.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			JOptionPane.showMessageDialog(null, "Information not saved");
+
+		}
+	}
+
+	/*
+	 * Create method to read from file
+	 */
+
+	public void read() {
```
Comment (rmit-s3675667-raj-shrestha) [inline src/main/FileService.java]: Make this method to return file content.

File: src/main/FileService.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+/**
+ * 
+ */
+package main;
```
Comment (rmit-s3675667-raj-shrestha) [inline src/main/FileService.java]: Add this class into different package.

**Comment by AliAla** [comment]
@rmit-s3675667-raj-shrestha 
Done 
I have changed them
Thanks

File: src/fileService/FileService.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+public class FileService {
+
+	private String fileName = "jMoSS.txt";
+
+	/*
+	 * Create method to write on file
+	 */
+	
+	// Parameters we can add them as we need 
+	public void write(String theaterLocation, String movieName, String date, String time) {
```
Comment (rmit-s3675667-raj-shrestha) [inline src/fileService/FileService.java]: file writing method needs to be generic. There are different types of data needs to be written into the file.

File: src/fileService/FileService.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+public class FileService {
+
+	private static String fileName = "jMoSS.txt";
+
+	/*
+	 * Create method to write on file
+	 */
+	
+	// Parameters we can add them as we need 
+	public static <T>void write(T object) {
```
Comment (rmit-s3675667-raj-shrestha) [inline src/fileService/FileService.java]: Generic means, not `generic` method. In general send string to write.

**Comment by AliAla** [comment]
Do you mean to be like :
public static String void write(String object)
??

File: src/fileService/FileService.java (hunk 5 above)
Comment (rmit-s3675667-raj-shrestha) [reply src/fileService/FileService.java]: `public void write(String txt)`

**Comment by rmit-s3675667-raj-shrestha** [comment]
code review done 👍

**Pull request merged**